### PR TITLE
Fix typos in README

### PR DESCRIPTION
**Pull request by rasa**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -330,3 +328,7 @@ Developer info
 [winsw-1.17]: https://github.com/kohsuke/winsw/milestones/winsw-1.17
 [winsw-1.17-beta.2]: https://github.com/kohsuke/winsw/releases/tag/1.17-beta.2
 [WinSW-2.0]: https://github.com/kohsuke/winsw/milestones/winsw-2.0
+[TerminateProcess]: http://msdn.microsoft.com/en-us/library/windows/desktop/ms686714(v=vs.85).aspx "TerminateProcess function"
+[this MSDN article]: http://msdn.microsoft.com/en-us/library/aa389390(VS.85).aspx "Create method of the Win32_Service class"
```
Comment (oleg-nenashev) [inline README.markdown]: I would more informative reference names

**Comment by rasa** [comment]
Agreed. I was just following the article's original text, but I didn't want to assume.

**Comment by rasa** [comment]
There ya go!

**Comment by oleg-nenashev** [comment]
Sorry, I've missed the second patch.
Thanks a lot for the docs review!

**Pull request merged**